### PR TITLE
[INVE-597] Fixed dashboard dragging overlay and some console errors.

### PR DESCRIPTION
**Pull request by NotHexadecimal**
Jira issue: https://sirensolutions.atlassian.net/browse/INVE-597

Related PRs:
- https://github.com/sirensolutions/kibi-internal/pull/12197

These changes prevent the overlay from remaining stuck on the map when releasing a dashboard quickly after starting dragging it (which only occurred when the dashboard being dragged didn't have a geo field). This PR also fixes a couple console errors and a typo.

### CODE REVIEW

File: public/lib/dragAndDroplayercontrol/DndLayerControl.js [hunk 1]
```diff
@@ -484,7 +484,7 @@ function getExtendedMapControl() {
           }
         }
       },
-      _source: ['geometrytype', 'spatial_path'],
+      _source: ['geometry', 'spatial_path'],
```
Comment (corrigancd) [inline public/lib/dragAndDroplayercontrol/DndLayerControl.js:487]: What was this change for?

Comment (NotHexadecimal) [reply public/lib/dragAndDroplayercontrol/DndLayerControl.js:487]: It solves a console error, not really sure what it does exactly, but the data from the query response had a `geometry.type` property and no `geometrytype` property, so I changed that and the console error disappeared, should that have been left as it was?

Comment (corrigancd) [reply public/lib/dragAndDroplayercontrol/DndLayerControl.js:487]: I think so, there was a change made to kibi-distrubution about a week ago. It's for stored layers. If you click in the top right of the map and click add layers, do things work with that change?

**Comment by johnnylopes** [comment]
It seems fine now. thanks, @NotHexadecimal 

If the user holds the button (on drag and drop) it will show the message...if the user releases it, it will not block the map.

map

**Pull request merged**